science/memesuite/Portfile

```diff
@@ -3,48 +3,72 @@
-                    http://alternate.meme-suite.org/meme-software/macports
+                    motifs in DNA, RNA and proteins.
+homepage            https://meme-suite.org
+master_sites        https://meme-suite.org/meme/meme-software/${version}
+distname            meme-${version}
+
+checksums           rmd160  a068ad5b3cc70149477d3b0d7f00e231a628f865 \
+                    sha256  bc6f693bf5d81778b4e2901c2164c35c68f42ed080e2401b6e22960bf039a272 \
+                    size    52764373
 
-checksums           rmd160  4a0b181454d136782e2fe2a5d0bce2f8363be478 \
-                    sha256  5bae8c2fa649bcafd6bd49dfe024d2e1d577cc802f9de2ba9dba3af05191093e \
-                    size    46161174
+set perl_vers       5.34
 
 depends_lib         port:expat \
                     port:gawk \
                     port:ghostscript \
                     port:gsed \
-                    port:perl5.28 \
-                    port:p5.28-file-which \
-                    port:p5.28-html-template \
-                    port:p5.28-html-tree \
-                    port:p5.28-json \
-                    port:p5.28-math-cdf \
-                    port:p5.28-xml-parser \
-                    port:p5.28-xml-simple \
-                    port:python37 \
-                    port:py37-gnureadline \
+                    port:perl${perl_vers} \
+                    port:p${perl_vers}-file-which \
+                    port:p${perl_vers}-html-template \
+                    port:p${perl_vers}-html-tree \
+                    port:p${perl_vers}-json \
+                    port:p${perl_vers}-math-cdf \
+                    port:p${perl_vers}-sys-info \
+                    port:p${perl_vers}-xml-parser \
+                    port:p${perl_vers}-xml-simple \
+                    port:python310 \
+                    port:py310-gnureadline \
                     port:zlib
 
 configure.args      --enable-build-libxml2 \
                     --enable-build-libxslt \
-                    --with-python=${prefix}/bin/python3.7 \
-                    --with-perl=${prefix}/bin/perl5.28 \
+                    --with-python=${prefix}/bin/python3.10 \
+                    --with-perl=${prefix}/bin/perl${perl_vers} \
                     --with-gs=${prefix}/bin/gs
 
+configure.cppflags-delete    -I${prefix}/include
+
+#
+#
+# Make serial version of MEME.
+variant serial description {Build MEME for serial mode.} {
+                    configure.args-append --enable-serial
+}
+
+# Make parallel version of MEME using MacPorts port of openmpi.
+variant parallel description {Build MEME for parallel mode.} {
+                    depends_lib-append port:openmpi
+                    configure.args-append --with-mpidir=${prefix}
+}
+
+notes "The MEME Suite is free for educational, research and non-profit purposes. Commercial users should obtain a license as described here: https://meme-suite.org/meme/doc/copyright.html."
+
 use_parallel_build  yes
```

Comment:
this is the default and can thus be removed

Reply:
I agree to this change.